### ISSUE TITLE
Pop!_OS 22.04 fails to install on Acer CB512 Chromebook

### ISSUE THREAD

**Issue by gallowglass** [pop-os/pop#3359]
**Distribution (run `cat /etc/os-release`):**
 
 22.04
 
**Issue/Bug Description:**

I tried to install Pop!_OS 22.04 onto a chromebook and it failed.  Pop booted, but the installer crashed after selecting language and locale.

**Steps to reproduce (if you know):**

1. Install coreboot firmware on chromebook.
2. Boot Pop!_OS from USB.
3. Select language and locale.

**Expected behavior:**

Installation would continue to completion.

**Other Notes:**

I have uploaded the logs from the failed installation attempts.

[failed_install_logs.tar.gz](https://github.com/user-attachments/files/16631034/failed_install_logs.tar.gz)

**Comment by leviport** [comment]
There aren't any installer logs in that archive. The log collector is not intended to collect installer logs.

**Comment by gallowglass** [comment]
Sorry about that.  I mistook the lines like

`Jun 15 01:21:18 pop-os io.elementary.installer.desktop[2691]: fatal runtime error: failed to initiate panic, error 5`

as something that might help.

Is there something I can do to help get this working, or is it not worth the effort?  (Installing Pop!_OS on chromebooks might not be something y'all are interested in pouring effort into and that would be totally understandable.)  If people are interested in pursuing this, I can collect more information, just tell me what you need.

**Comment by leviport** [comment]
When the installer fails, it tells you where that installer.log file is stored.

**Comment by gallowglass** [comment]
Okay.  I'll try again and see if I can find that file.

**Comment by gallowglass** [comment]
Alas, I got the installation to work.  Which is good for me, but bad for figuring out what went wrong before.  

The thing I did differently this time is that after the installer died as above, used fdisk to delete the **13** partitions ChromeOS had on the disk, then opened the installer again.  So there is apparently something about having an obscene number of partitions that was throwing the installer for a loop.
